test/client_helper_test.rb

```diff
@@ -77,6 +77,13 @@
       html = recaptcha_tags(stoken: false)
       html.must_include "<div class=\"g-recaptcha\" data-sitekey=\"0000000000000000000000000000000000000000\"></div>"
     end
+
+    it "should raise an error if secure_token is called without a private_key" do
```

Comment:
how about the simpler:
`it "raises if secure_token is called without a private_key" do`